### PR TITLE
Initial Setup - WIP

### PR DESCRIPTION
**Pull request by kerrd89**
using the backend server provided, create react app, and axios, my prework is ready for review.  I created a quiz component, a questions component, and an answers component.  

I am wondering about the server updating... do we need a 'shouldComponentUpdate()' and 'didComponentUpdate()'?

### CODE REVIEW

**Comment by kerrd89** [comment]
@brittanystoroz and @stevekinney

File: src/Answers.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './App.css';
+
+const Answers = ({answers}) => {
+  let answersHTML = answers.map((answer, index) => {
+    return (
+      <p className="quiz-answer" key={index}>{answer.title}</p>
```
Comment (brittanydionigi) [inline src/Answers.js]: I would maybe make each answer a `label` element with a `radio` button so that they are selectable.

**Review by brittanydionigi — CHANGES_REQUESTED** [review]
Tiny change otherwise r+

Don't wonder about the server updating, that won't be an issue.

File: src/Answers.js (hunk 1 above)
Comment (kerrd89) [reply src/Answers.js]: I created an input field with the type of radio and the ID of index.  I added a label for that ID and put the answers into an li so I would have a single dom element wrapper.  This better?

**Comment by brittanydionigi** [comment]
Radio elements need a `name` attribute to help identify that they are part of a group. See example [here](http://www.w3schools.com/html/tryit.asp?filename=tryhtml_input_radio).

The name should be unique per question. i.e. all of the answer elements in question one might have a `name` attribute equal to `answer-group-1`.

File: src/Answers.js (hunk 1 above)
Comment (kerrd89) [reply src/Answers.js]: thanks for the link!  I think this is what you are looking for.  I added this to the input: 

`name={'answer-group-'+index}`

**Pull request merged**